ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dhcp-servers.j2

```diff
@@ -18,6 +18,34 @@
 {%         if dhcp_server.dns_domain_name_ipv6 is arista.avd.defined %}
    dns domain name ipv6 {{ dhcp_server.dns_domain_name_ipv6 }}
 {%         endif %}
+{%         if dhcp_server.dns_server.ipv4_servers is arista.avd.defined %}
+{%             set dns_servers_cli = namespace(cli="dns server ipv4") %}
+{%             for server in dhcp_server.dns_server.ipv4_servers  %}
+{%                 set dns_servers_cli.cli = dns_servers_cli.cli ~ " " ~ server %}
+{%             endfor %}
+   {{ dns_servers_cli.cli }}
+{%         endif %}
+{%         if dhcp_server.dns_server.ipv6_servers is arista.avd.defined %}
+{%             set dns_servers_cli = namespace(cli="dns server ipv6") %}
+{%             for server in dhcp_server.dns_server.ipv6_servers  %}
+{%                 set dns_servers_cli.cli = dns_servers_cli.cli ~ " " ~ server %}
+{%             endfor %}
+   {{ dns_servers_cli.cli }}
+{%         endif %}
+{%         if dhcp_server.tftp_server.file.ipv4_file is arista.avd.defined %}
+{%             set tftp_file_cli = namespace(cli="tftp server file ipv4") %}
+{%             for file in dhcp_server.tftp_server.file.ipv4_file | arista.avd.natural_sort %}
+{%                 set tftp_file_cli.cli = tftp_file_cli.cli ~ " " ~ file %}
+{%             endfor %}
+   {{ tftp_file_cli.cli }}
```

Comment:
Because we are joining a list of strings, we cannot validate the length using jsonschema(maybe we can), but I think the validation of the length belongs here.  Can we raise an exception here if the total length >255?  https://datatracker.ietf.org/doc/html/rfc2132#section-9.5. Failing early would be useful here.

Reply:
We do not validate data in the templates. I am failing to understand what the issue is you wish to catch.

Reply:
If we want to validate the length of the file name, I feel like we should be doing that in eos_designs.

Reply:
We have logic in the template which concatenates a list of server strings.  That concatenated list cannot be greater than 255.  If the input of the model was just an opaque string, instead of a list, that was used to render the output, then the data could be validated via jsonschema, just like other values. If the logic to build the string is in the template, the equivalent validation of the length, that we would normally get from jsonschema should be added to the template.  If validation should not be in the template, would it not be better for the model to just be based on the final opaque string and let jsonschema validate the final unmanipulated max length?  Why would we want to potentially propagate a possible problem to eos_validate_state or later to catch?

Reply:
agreed. Let's change the model to a string, and if the use needs to enter multiple values they can.